pandas/core/groupby/generic.py

```diff
@@ -386,7 +386,7 @@ def _wrap_aggregated_output(
         result = self._wrap_series_output(
             output=output, index=self.grouper.result_index
         )
-        return self._reindex_output(result)._convert(datetime=True)
+        return self._reindex_output(result)
```

Comment:
Do you have an explanation of why this is not needed anymore?

Reply:
No, just an observation that we dont have any tests that break without it.